### DISCUSSION TITLE
配置连接成功后无法上网

### DISCUSSION THREAD

**Discussion by LittleXu** [Qv2ray/qv2ray.github.io#127] [Q&A] [answered]
日志大概是这样
V2Ray 4.36.2 (V2Fly, a community-driven edition of V2Ray.) Custom (go1.16 darwin/amd64)
A unified platform for anti-censorship.
2021/04/01 19:18:40 [Info] github.com/v2fly/v2ray-core/v4/main/jsonem: Reading config: /Users/a111/Library/Preferences/qv2ray/generated/config.gen.json
2021/04/01 19:18:40 [Warning] github.com/v2fly/v2ray-core/v4: V2Ray 4.36.2 started
2021/04/01 19:18:41 127.0.0.1:49938 accepted tcp:127.0.0.1:0 [QV2RAY_API]
2021/04/01 19:18:45 127.0.0.1:49951 accepted //note.youdao.com:443 [DIRECT]
2021/04/01 19:18:45 127.0.0.1:49953 accepted //p44-caldav.icloud.com.cn:443 [DIRECT]
2021/04/01 19:18:45 127.0.0.1:49958 accepted //setup.icloud.com:443 [wrbzptzrckpvvuf]
2021/04/01 19:18:50 [Warning] [810484986] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:18:59 127.0.0.1:50011 accepted //www.google.com:443 [wrbzptzrckpvvuf]
2021/04/01 19:18:59 127.0.0.1:50018 accepted //suggestion.baidu.com:443 [DIRECT]
2021/04/01 19:19:01 127.0.0.1:50025 accepted //www.google.com.hk:443 [wrbzptzrckpvvuf]
2021/04/01 19:19:01 127.0.0.1:50026 accepted //www.google.com.hk:443 [wrbzptzrckpvvuf]
2021/04/01 19:19:01 127.0.0.1:50027 accepted //www.gstatic.com:443 [wrbzptzrckpvvuf]
2021/04/01 19:19:03 [Warning] [690791023] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:19:03 127.0.0.1:50050 accepted //www.google.com:443 [wrbzptzrckpvvuf]
2021/04/01 19:19:05 [Warning] [928433451] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:19:05 127.0.0.1:50063 accepted //www.google.com.hk:443 [wrbzptzrckpvvuf]
2021/04/01 19:19:05 [Warning] [2436082882] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:19:05 127.0.0.1:50064 accepted //www.google.com.hk:443 [wrbzptzrckpvvuf]
2021/04/01 19:19:05 [Warning] [13760518] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:19:05 127.0.0.1:50067 accepted //www.gstatic.com:443 [wrbzptzrckpvvuf]
2021/04/01 19:19:08 [Warning] [1134655342] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:19:09 [Warning] [3331125852] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:19:09 [Warning] [3471367075] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:19:09 [Warning] [387763824] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:19:10 127.0.0.1:50101 accepted //www.google.com.hk:443 [wrbzptzrckpvvuf]
2021/04/01 19:19:10 127.0.0.1:50102 accepted //www.google.com.hk:443 [wrbzptzrckpvvuf]
2021/04/01 19:19:10 127.0.0.1:50103 accepted //www.gstatic.com:443 [wrbzptzrckpvvuf]
2021/04/01 19:19:15 [Warning] [3654048199] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:19:15 127.0.0.1:50135 accepted //www.gstatic.com:443 [wrbzptzrckpvvuf]
2021/04/01 19:19:15 [Warning] [711170635] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:19:15 127.0.0.1:50137 accepted //www.google.com.hk:443 [wrbzptzrckpvvuf]
2021/04/01 19:19:15 [Warning] [4231572403] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:19:15 127.0.0.1:50139 accepted //www.google.com.hk:443 [wrbzptzrckpvvuf]
2021/04/01 19:19:19 [Warning] [3832450255] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:19:19 [Warning] [293311224] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:19:19 [Warning] [23700145] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:19:20 127.0.0.1:50170 accepted //clients4.google.com:443 [wrbzptzrckpvvuf]
2021/04/01 19:19:24 [Warning] [2315634829] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:19:24 127.0.0.1:50197 accepted //clients4.google.com:443 [wrbzptzrckpvvuf]
2021/04/01 19:19:29 [Warning] [2822360324] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:20:12 127.0.0.1:50364 accepted //www.google.com.hk:443 [wrbzptzrckpvvuf]
2021/04/01 19:20:12 127.0.0.1:50367 accepted //safebrowsing.googleapis.com:443 [wrbzptzrckpvvuf]
2021/04/01 19:20:16 [Warning] [1882434555] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:20:16 127.0.0.1:50393 accepted //www.google.com.hk:443 [wrbzptzrckpvvuf]
2021/04/01 19:20:16 [Warning] [3795532217] github.com/v2fly/v2ray-core/v4/app/proxyman/outbound: failed to process outbound traffic > github.com/v2fly/v2ray-core/v4/proxy/vless/outbound: failed to find an available destination > github.com/v2fly/v2ray-core/v4/common/retry: [github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial WebSocket > github.com/v2fly/v2ray-core/v4/transport/internet/websocket: failed to dial to (ws://node-ja1.freezone.cloud:443/kaelws): 400 Bad Request > websocket: bad handshake] > github.com/v2fly/v2ray-core/v4/common/retry: all retry attempts failed
2021/04/01 19:20:16 127.0.0.1:50399 accepted //safebrowsing.googleapis.com:443 [wrbzptzrckpvvuf]

**Comment by DuckSoft** [comment]
又是 CloudFlare 的，被墙了吧

**Comment by LittleXu** [comment]
CloudFlare 是什么？

**Comment by DuckSoft** [comment, marked as answer]
ws://node-ja1.freezone.cloud:443/kaelws
是不是忘记开tls了？

正常情况应该是wss://node-ja1.freezone.cloud:443/kaelws这样

**Reply by LittleXu** [reply]
非常感谢，开启tls之后可以了
